pkgs/development/tools/misc/gdb/default.nix

```diff
@@ -27,6 +27,7 @@ let
 in
 
 assert pythonSupport -> python3 != null;
+assert lib.asserts.assertMsg (stdenv.targetPlatform.system != "aarch64-darwin") "GDB does not support aarch64-darwin as a target";
```

Comment:
Can't this use meta.badPlatforms?

Reply:
I don't believe so; the code for checking against `platforms` and `badPlatforms` appears to use `hostPlatform`:
https://github.com/NixOS/nixpkgs/blob/40222c1c96c73fdb47070417a5ae34ffb314ad35/pkgs/stdenv/generic/check-meta.nix#L104-L105

Is there a better way to advertise support for a `hostPlatform`, but not a `targetPlatform`?